ui/InstrumentView.py

```diff
@@ -0,0 +1,146 @@
+from PySide2.QtWidgets import QWidget, QVBoxLayout
```

Comment:
I think only the generated UI files and resources like icons etc. should be in this file. Just my opinion, but i think we should move these out into the `nexus_constructor/` directory?